### ISSUE TITLE
Make a branch and ubuntu packages for indigo and jade

### ISSUE THREAD

**Issue by dogoepp** [youbot/youbot_driver_ros_interface#11]
Hi,

I guess it would cost little to make a branch for ros indigo and make a ubuntu package from it, since it is already done for hydro.

I can tell that there is no need for change between the hydro version and the indigo one, as our Youbot is on indigo.

**Comment by frederikhegger** [comment]
The Indigo branch is there now. And I guess the Jade branch will follow soon.

**Comment by dogoepp** [comment]
Indeed. What about Jade, Kinetic and Lunar ? I guess there is little change to do, but it would be nice to have the youbot packages be part of the ROS distribution.
